Resources/translations/sub_items.en.xliff

```diff
@@ -7,10 +7,15 @@
     </header>
     <body>
       <trans-unit id="35e03083bfdfafc073cf343c74385bec3945b4ac" resname="bulk_delete.error.message">
-        <source>You do not have permission to delete at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</source>
-        <target state="new">You do not have permission to delete at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.</target>
+        <source><![CDATA[%notDeletedCount% of the %totalCount% selected item(s) could not be deleted due to lack of your user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></source>
+        <target state="new"><![CDATA[%notDeletedCount% of the %totalCount% selected item(s) could not be deleted due to lack of your user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></target>
```

Comment:
```suggestion
        <target state="new"><![CDATA[%notDeletedCount% of the %totalCount% selected item(s) could not be deleted because you do not have proper user permissions. <u><a class='ez-notification-btn ez-notification-btn--show-modal'>Click here for more information.</a></u><br>Please contact your Administrator to obtain permissions.]]></target>
```